### PR TITLE
Base code framework for HWCC

### PR DESCRIPTION
**Pull request by Ashughorla**
This implementation is done based on https://github.com/metal3-io/metal3-docs/pull/60.

### CODE REVIEW

File: config/certmanager/certificate.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```
Comment (dhellmann) [inline config/certmanager/certificate.yaml:1]: What are the files in the certmanager directory for?

**Review by dhellmann — COMMENTED** [review]
This all looks like the generated boilerplate, which is a great way to start. I had 1 question about some of the files that I don't recognize that I would like to understand before we merge, but otherwise it looks good.

File: config/certmanager/certificate.yaml (hunk 1 above)
Comment (digambar15) [reply config/certmanager/certificate.yaml:1]: Cert-manager is used for web hooks, currently its disabled if we want to use webhooks in the future, then we can enable it. It's generated by default via Kube-builder.

**Comment by digambar15** [comment]
Looks good

**Comment by dhellmann** [comment]
We should set up prow to handle merges in this repo. @russellb are there instructions for doing that?

**Comment by russellb** [comment]
It should be automatic because it's configured against the whole github org.  An OWNERS file needs to be created, though.

**Comment by dhellmann** [comment]
@Ashughorla could you create an OWNERS file, please, either in a separate PR or as another patch in this PR?

**Comment by Ashughorla** [comment]
@dhellmann Added OWNERS file as another patch in this PR.

**Comment by russellb** [comment]
Once the owners file merges, prow should pick up future PRs.

**Pull request merged**